src/codegen/codegen_neuron_cpp_visitor.cpp

```diff
@@ -754,7 +805,30 @@ void CodegenNeuronCppVisitor::print_mechanism_global_var_structure(bool print_in
     }
 
     if (info.table_count > 0) {
-        throw std::runtime_error("Not implemented, global table count.");
+        // basically the same code as coreNEURON uses
```

Comment:
As mentioned, this taken verbatim from coreNEURON. I could move this to `CodegenCppVisitor` if needed, since the code is identical (if there's no potential issues that could arise with other visitors).

Reply:
@JCGoran : IMO, if there is a common code then having it in the top level/base visitor would be helpful than having two copies.

Reply:
Most of the common code was merged in #1251